python/modules/BtagSFWeightProducer.py

```diff
@@ -62,7 +62,7 @@ def beginFile(self, inputFile, outputFile, inputTree, wrappedOutputTree):
             del self.h_eff_udsg
 
         if self.FastSim:
-            sampleName = os.path.splitext(os.path.basename(inputFile.GetName()))[0]
+            sampleName = os.path.splitext(os.path.basename(inputFile.GetName()))[0].split("_split")[0]
```

Comment:
what is this fixing?

Reply:
This is used for fastsim signals needed to be splited when the file is too large and run for over 2 days. The splitting script is also added as "SplitNanoAOD.py"